timescaledb/how-to-guides/multinode-timescaledb/multinode-administration.md

```diff
@@ -0,0 +1,257 @@
+creating roles. The role needs to altered locally and on the data
+nodes in two separate steps. For example, to add the `CREATEROLE`
+attribute to a role do:
+
+```sql
+ALTER ROLE alice CREATEROLE;
+CALL distributed_exec($$ ALTER ROLE alice CREATEROLE; $$);
+```
+
+## Managing distributed databases
+A distributed database can contain both distributed and
+non-distributed objects. In general, when a command is issued to alter
+a distributed object it applies to all nodes that have that object (or
+a part of it).
+
+However, in some cases settings *should* be different depending on
+node, because nodes might be provisioned differently (having, e.g.,
+varying levels of CPU, memory, and disk capabilities) and the role of
+the access node is different from a data node's.
+
+This section describes how and when commands on distributed objects
+are applied across all data nodes when executed from within a
+distributed database.
+
+### Altering a distributed database
+An [`ALTER DATABASE`][alter-database] command is only applied locally on the access
+node. There are several reasons why TimescaleDB does *not* apply this
+command across all data nodes.
+
+First, a distributed database supports having different database names
+on each data node. Therefore, renaming a distributed database on the
+access node is a local operation that applies only to the access node
+database. Note that when renaming a data node database the
+configuration of the data node must also be updated on the access
+node.
+
+
+Second, database-level configuration parameters should often be
+different across nodes. For example:
+
+```sql
+ALTER DATABASE mydatabase SET max_parallel_workers TO 12;
+```
+
+is a setting that might differ depending on the CPU capabilities of
+the node.
+
+Third, a database option like:
+
+```sql
+ALTER DATABASE mydatabase WITH CONNECTION_LIMIT 10;
+```
+
+is also, in most cases, different on an access node compared to its
+data nodes.
+
+### Dropping a distributed database
+When issuing the command to drop a distributed database on the access
+node, it will *not* automatically drop the corresponding databases on
+the data nodes. Instead, it is necessary to connect directly to each
+data node and drop the databases locally.
+
+There are several reasons for why a distributed database is not
+automatically dropped across all nodes. 
+
+First, the information about data nodes lives within the distributed
+database on the access node, but it is not possible to read it when
+executing the drop command since it cannot be issued when connected to
+the database.
+
+Second, in case a data node has permanently failed, one wants to be
+able to drop a database even if one or more data nodes are not
+responding.
+
+Third, it is often good to leave the data intact on a data node out of
+precaution. For example, one might want to backup a data node even
+after a database was dropped on the access node.
```

Comment:
```suggestion
A distributed database is not automatically dropped across all nodes, 
because the information about data nodes lives within the distributed 
database on the access node, but it is not possible to read it when 
executing the drop command since it cannot be issued when 
connected to the database.

Additionally, if a data node has permanently failed, you need to be able 
to drop a database even if one or more data nodes are not responding.

It is also good practice to leave the data intact on a data node if 
possible. For example, you might want to back up a data node even 
after a database was dropped on the access node.
```